src/odemis/acq/test/path_test.py

```diff
@@ -48,11 +48,12 @@
 CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
 # Test for the different configurations
 SPARC_CONFIG = CONFIG_PATH + "sim/sparc-sim.odm.yaml"
-SECOM_CONFIG = CONFIG_PATH + "sim/secom2-sim.odm.yaml"
```

Comment:
For some reason, the backend would not start without this change.

Reply:
Well, as you can guess, that's not the proper way to fix it ;-) Let's have a look on your computer what's preventing the secom2 simulator from starting. Might need to install or configure something extra.

Reply:
Still need to revert this line. `secom-sim` ought to work fine.